src/Flex.js

```diff
@@ -1,57 +1,22 @@
 
 import React from 'react'
-import assign from 'object-assign'
-import Base from './Base'
-import config from './config'
+import Reflex from './Reflex'
 
 /**
  * Creates a flexbox context to control layout of children.
  */
 
-const Flex = ({
-  wrap,
-  column,
-  align,
-  justify,
-  auto,
-  ...props
-}, { reflexbox }) => {
-  let display = 'flex'
-  const { breakpoints } = { ...config, ...reflexbox }
+const Flex = ({ is = 'div', className, ...props }) => {
+  const Base = Reflex(is)
```

Comment:
this sounds like a bad idea performance-wise because react short-circuits reconciliation if the types of the new and old nodes are different (https://facebook.github.io/react/docs/reconciliation.html#different-node-types). So basically every time a `Flex` component re-renders the whole sub-tree will be discarded